tests/integration/cass_integration_test.go

```diff
@@ -0,0 +1,150 @@
+package integration
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/k8ssandra/k8ssandra/tests/integration/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	chartName         = "cass-operator"
+	chartPath         = "../../charts/k8ssandra/charts/cass-operator"
+	configPath        = "C:\\Users\\Jeff Banks\\.kube\\config"
+	crdDefinitionYaml = "../../charts/k8ssandra/charts/cass-operator/crds/customresourcedefinition.yaml"
+	crdResourceName   = "cassandradatacenters.cassandra.datastax.com"
+
+	dc1Name          = "dc1"
+	dc1Yaml          = "../../cassandra/example-cassdc-minimal.yaml"
+	defaultNamespace = "default"
+
+	namespacePrefix = "test-ns-cassop"
+	releaseName     = "test-release-cassop"
+	testName        = "Helm Chart cass-cluster"
+)
+
+// Setup performs namespace lookup having test related resources to be cleaned.
+func setup(t *testing.T) (string, *k8s.KubectlOptions, *helm.Options) {
+
+	t.Parallel()
+	var setupWaiter sync.WaitGroup
+	var namespace string
+
+	setupWaiter.Add(1)
+	go func() {
+		defer setupWaiter.Done()
+		namespace = util.GenerateNamespaceName(namespacePrefix)
+	}()
+	setupWaiter.Wait()
+
+	kubeOptions := k8s.NewKubectlOptions("", "", namespace)
+	helmOptions := &helm.Options{KubectlOptions: kubeOptions}
+
+	var cleanupWaiter sync.WaitGroup
+	cleanup(t, helmOptions, &cleanupWaiter)
+	cleanupWaiter.Wait()
+
+	return namespace, kubeOptions, helmOptions
+}
+
+//
+func teardown(t *testing.T, helmOptions *helm.Options) {
+	var cleanupWaiter sync.WaitGroup
+	cleanup(t, helmOptions, &cleanupWaiter)
+	cleanupWaiter.Wait()
+}
+
+// TestCassOperator performs basic installation of cass-operator.
+func TestCassOperator(t *testing.T) {
+
+	// Setup
+	namespace, kubeOptions, helmOptions := setup(t)
+
+	assert.NotNil(t, namespace)
+	assert.NotNil(t, kubeOptions)
+	assert.NotNil(t, helmOptions)
+
+	util.CreateNamespace(t, helmOptions)
+	util.ApplyAnnotation(t, helmOptions, releaseName)
+	
+	isInstalled := installCassOperator(t, helmOptions)
+	require.True(t, isInstalled)
+
+	lookupResult := util.LookupCRDByName(t, helmOptions, "cassandradatacenters.cassandra.datastax.com")
+	require.Equal(t, "customresourcedefinition.apiextensions.k8s.io/cassandradatacenters.cassandra.datastax.com", lookupResult)
```

Comment:
It would be good to verify that the cass-operator Deployment is ready. For example, here is my install,

```
 helm ls
NAME     	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART                   	APP VERSION
demo     	default  	1       	2020-11-17 20:11:06.222593 -0500 EST	deployed	k8ssandra-cluster-0.10.0	3.11.7
k8ssandra	default  	1       	2020-11-17 10:09:53.92589 -0500 EST 	deployed	k8ssandra-0.9.0         	3.11.7
```

Here are the deployments,

```
$ kubectl get deployments
NAME                                 READY   UP-TO-DATE   AVAILABLE   AGE
cass-operator                        1/1     1            1           24h
demo-grafana-operator-k8ssandra      1/1     1            1           14h
demo-reaper-k8ssandra                1/1     1            1           14h
demo-reaper-operator-k8ssandra       1/1     1            1           14h
grafana-deployment                   1/1     1            1           14h
k8ssandra-kube-prometheus-operator   1/1     1            1           24h
```
And now let's look at the cass-operator Deployment,

```
$ kubectl get deployment cass-operator -o yaml
...
status:
  availableReplicas: 1
  conditions:
  - lastTransitionTime: "2020-11-17T15:10:01Z"
    lastUpdateTime: "2020-11-17T15:10:08Z"
    message: ReplicaSet "cass-operator-86d4dc45cd" has successfully progressed.
    reason: NewReplicaSetAvailable
    status: "True"
    type: Progressing
  - lastTransitionTime: "2020-11-17T23:05:40Z"
    lastUpdateTime: "2020-11-17T23:05:40Z"
    message: Deployment has minimum availability.
    reason: MinimumReplicasAvailable
    status: "True"
    type: Available
  observedGeneration: 1
  readyReplicas: 1
  replicas: 1
  updatedReplicas: 1
```
We want to make sure that `.status.replicas` matches `.status.readyReplicas`.